### PR TITLE
fix read_sc_all_alarms_telecare with incorrect format in period

### PR DESCRIPTION
**Pull request by lizihao-anu**

### CODE REVIEW

**Review by Moohan — DISMISSED** [review]
Looks good, fixes the issue and is nice and general so should also work into the future!

**Comment by lizihao-anu** [comment]
> Looks good, fixes the issue and is nice and general so should also work into the future!

Thanks. Just waiting for the final test run to make sure that there is no further issue.

**Comment by lizihao-anu** [comment]
Be ready to merge now!

File: R/process_sc_all_alarms_telecare.R
Comment (Moohan) [inline R/process_sc_all_alarms_telecare.R:1]: I've fixed this issue separately in #810, so can you revert this change please?

Comment (lizihao-anu) [reply R/process_sc_all_alarms_telecare.R:1]: Sure!

**Comment by Moohan** [comment]
> @check-spelling-bot apply [updates](https://github.com/Public-Health-Scotland/source-linkage-files/actions/runs/5877228247/attempts/1).

**Review by Moohan — APPROVED** [review]
